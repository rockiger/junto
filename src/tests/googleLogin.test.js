import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';

import GoogleLogin from '../components/googleLogin';

describe('GoogleLogin', () => {
    const props = {
        setIsSigningIn: (isSignedIn) => {},
    }
    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(
            <GoogleLogin {...props} />, div);
        ReactDOM.unmountComponentAtNode(div);
    });

    test('has a valid snapshot', () => {
        const component = renderer.create(
            <GoogleLogin {...props} />
        );
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});