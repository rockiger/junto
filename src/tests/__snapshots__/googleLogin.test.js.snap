// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GoogleLogin has a valid snapshot 1`] = `
<button
  className="action"
  id="authorize_button"
  onClick={[Function]}
>
  Login with Google
</button>
`;
